torch/optim/swa_utils.py

```diff
@@ -301,8 +301,8 @@ def update_bn(
     model: Module,
     device: Optional[Union[int, torch.device]] = None,
 ):
-    r"""Update BatchNorm running_mean, running_var buffers in the model.
-
+    """
+    Update BatchNorm running_mean, running_var buffers in the model.
```

Comment:
I don't think this will do anything! 

Can you try adding a line before 308? (Args:...)

Reply:
Sorry! I just noticed my last comment never sent cuz it was in pending mode :C

Reply:
@janeyx99 :) ah, thanks and no worries at all. Yes, let me follow your suggestion, thank you so much for it. I I will update the PR soon. Also, is `make html` the best way to run tests locally? I am not sure why but somehow, when I run it, I don't see the error that the build gives.